aws_lambda_powertools/logging/utils.py

```diff
@@ -0,0 +1,30 @@
+import logging
+from typing import List, Optional, TypeVar
+
+from .logger import Logger
+
+PowertoolsLogger = TypeVar("PowertoolsLogger", bound=Logger)
+
+
+def copy_config_to_registered_loggers(
+    source_logger: PowertoolsLogger, exclude: Optional[List[str]] = None, include: Optional[List[str]] = None
+):
+    root_loggers = [
```

Comment:
Use a more explicit name to avoid confusion in the future with actual root logger.

```suggestion
    registered_loggers = [
```

Reply:
Done, also modified corresponding internal methods